src/components/App.test.js

```diff
@@ -3,6 +3,6 @@ import App from './App';
 
 describe('App', () => {
   it('renders without crashing', () => {
-    renderWithRedux(<App />);
+    let wrapper = global.renderWithRedux(<App />);
```

Comment:
Probably best to scrap any testing related code from this PR. Testing stuff can live in a separate one.